src/core/mesh/qgstriangularmesh.h

```diff
@@ -85,6 +85,10 @@ class CORE_EXPORT QgsTriangularMesh
      */
     int faceIndexForPoint( const QgsPointXY &point ) const ;
```

Comment:
remove?

Reply:
I don't know, maybe there are advantages to use GEOS for some context ? Is the first version is used elsewhere ? My QtCreator doesn't find another use...